### PR TITLE
BD-3089 Create `includes` file for metrics

### PR DESCRIPTION
**Pull request by rachel-feinberg**
### Summary

My PR creates an includes file that contains metrics definitions, and inserts the include tag throughout the Docs site.

### Related PRs, issues, or features (optional)

[BD-3089](https://jira.braze.com/browse/BD-3089)

### Feature release date (optional)

- N/A

### Contributor checklist

- [ ] I confirm that my PR meets the following:
    - I signed the [Contribution License Agreement (CLA)](https://www.braze.com/docs/cla).
    - My style and voice follows the [Braze Style Guide](https://docs.google.com/document/u/2/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub).
    - My content contains correct spelling and grammar.
    - All links are working correctly.
    - If I renamed or moved a file or directory, I set up [URL redirects](https://www.braze.com/docs/contributing/content_management/redirecting_urls) for each file.
    - If I updated or replaced an image, I did not remove the original image file from the repository. (For more information, see [Updating an image](https://www.braze.com/docs/contributing/content_management/images/#updating-an-image)).
    - If my PR is related to a paid SKU, third party, SMS, AI, or privacy, I have received written approval from Braze Legal.

### Submitting for review

If your PR meets the above requirements, select **Ready for review**, then add a reviewer:
- **Non-Braze contributors:** Add `braze-inc/docs-team` as the reviewer.
- **Braze employees:** Have any relevant subject matter experts (like engineers or product managers) review your work first, then add the [tech writer assigned to your specific vertical](https://confluence.braze.com/pages/viewpage.action?pageId=75039896). If you're not sure which tech writer to add, you can add `braze-inc/docs-team` instead. 

Thanks for contributing! We look forward to reading your work.

### CODE REVIEW

**Review by bre-fitzgerald — APPROVED** [review]
Did a spot check of most of the pages and looks like everything is rendering as expected. Lgtm!

File: _docs/_user_guide/message_building_by_channel/email/reporting_and_analytics/analytics_glossary.md [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
   - email
+---
 
+<style>
+  .calculation-line {
+    color: #76848C;
+    font-size: 14px;
+  }
+</style>
+=======
```
Comment (lydia-xie) [inline _docs/_user_guide/message_building_by_channel/email/reporting_and_analytics/analytics_glossary.md:19]:
```suggestion
---

<style>
  .calculation-line {
    color: #76848C;
    font-size: 14px;
  }
</style>
=======
```
@rachel-feinberg I think this area might've broken [the page](https://braze-docs-gikztwry4-braze.vercel.app/docs/user_guide/message_building_by_channel/email/reporting_and_analytics/analytics_glossary)
Screenshot 2024-10-15 at 9 22 18 AM

File: _includes/campaign_analytics.md [hunk 2]
```diff
@@ -197,6 +235,75 @@ Your _Messages Sent_ refers to Content Cards available to be seen, while _Unique
...
+            <td class="no-split"><a href="/docs/user_guide/data_and_analytics/report_metrics/#hard-bounce">Hard Bounce</a></td>
+            <td class="no-split">{% multi_lang_include metrics.md metric='Hard Bounce' %}</td>
+        </tr>
+        <tr>
+            <td class="no-split"><a href="/docs/user_guide/data_and_analytics/report_metrics/#soft-bounce">Soft Bounce</a></td>
+            <td class="no-split">{% multi_lang_include metrics.md metric='Soft Bounce' %}</td>
+        </tr>
+    </tbody>
+</table>
+=======
```
Comment (lydia-xie) [inline _includes/campaign_analytics.md:306]:
```suggestion
```
Suggestion to remove this, see https://braze-docs-gikztwry4-braze.vercel.app/docs/user_guide/message_building_by_channel/email/reporting_and_analytics/email_reporting/#email-performance:~:text=%3D%3D%3D%3D%3D%3D%3D%20Here%20are%20some,Analytics%20Glossary.

**Review by lydia-xie — CHANGES_REQUESTED** [review]
Left 2 comments for your review!

File: _docs/_user_guide/message_building_by_channel/email/reporting_and_analytics/analytics_glossary.md (hunk 1 above)
Comment (rachel-feinberg) [inline _docs/_user_guide/message_building_by_channel/email/reporting_and_analytics/analytics_glossary.md:19]:
```suggestion
```

**Pull request merged**